clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/EcsProvider.java

```diff
@@ -72,4 +74,17 @@ public String getProviderName() {
     return agents;
   }
 
+
+  public void synchronizeHealthAgents() {
```

Comment:
I am confused by this method.  It's called `synchronize`, but it doesn't synchronize.  It just modifies the `healthAgents` field

Reply:
Following suit with the AWS module.

See `synchronizeHealthAgents` in `AwsProvider` class.